### DISCUSSION TITLE
What are malformed emails?

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#47] [Q&A] [answered]
Hi! My list has at present 32 "malformed" emails that seem, on close inspection, to look just fine. What's the deal with these? What happens to them, are they excluded from mailouts?

**Comment by krangor** [comment]
(Apologies for posting this in the wrong place... I wasn't sure if it was an issue or a question!)

**Reply by jmduke** [reply]
(No worries! It honestly does not matter that much, I just wanted to be a little clean-freak with it. Writing up your answer now..)

**Comment by jmduke** [comment, marked as answer]
I'll get this added to the docs as well, but:

- Yes, malformed subscribers are excluded from mailouts.
- While on the dashboard they _look_ well-formed, after looking at your 32 the reason they're flagged is because they have some hidden trailing whitespace. The email is not actually `patricia@gmail.com` (to take an example), it's `patricia@gmail.com\x00&nbsp;` — there's some hidden characters.
- Additionally, these subscribers look like they came from folks spamming your comments. Those hidden characters are a good tell for bots, since it's not something a human would ever type up.
- I'll try to figure out a way to surface that kind of thing in the dashboard (maybe a notice saying "this email address contains hidden characters! sus!") because your confusion is definitely understandable.

**Comment by krangor** [comment]
That makes sense, as they are otherwise very innocuous-looking... perhaps
suspiciously so! Simple first name + gmail is almost impossible to get.
